lang/en/texts/index-pro.html

```diff
@@ -0,0 +1,39 @@
+<h1>Open Food Facts for producers</h1>
+
+<div class="row hide-when-logged-in">
+
+<div class="medium-12 columns">
+
+<p>A completely free platform to allow manufacturers to easily manage the data and photos of their products on Open Food Facts.</p>
+
+<h2>Make your products visible in more than 100 apps and services</h2>/
+
+<p>Publish your products data on Open Food Facts and participate in the global movement for transparency on food products.</p>
+
+<ul>
+	<li>Your data is complete, correct and up-to-date.</li>
+	<li>Add extra information like traceability, a link to the product page on your site, and customer service contact info.</p>
+</ul>
+
+<h2>Easily adop the Nutri-Score</h2>
+
+<p>The Nutri-Score is automatically computed from the nutrition facts.</p>
+
+<ul>
+	<li>Understand the Nutri-Score calculation with all its components.</li>
+	<li>Verify the Nutri-Score if you have already calculated it.</li>
+</ul>
+
+<h2>Find opportunities to improve the quality of your products</h2>
+
+<p>We automatically analyze the data of your products and compare them to similar products.</p>
+
+<ul>
+	<li>Significantly improve the Nutri-Score with a reasonable reduction in sugar, salt or saturated fatty acides.</li>
+	<li>Detect products with a lower nutritional profile than similar products on the market.</li>
```

Comment:
```suggestion
	<li>Detect products with a better nutritional profile than similar products on the market.</li>
```

Reply:
It's the reverse, we want to detect products with a lower nutritional profile, so that we can improve them.